packages/plugin/webview/src/lib/connector.ts

```diff
@@ -24,11 +27,16 @@ export class WebviewConnector implements ClientConnector {
   source: { postMessage: (message: any, origin?: string) => void }
   origin: string
   isVscode: boolean
+  isTheia: boolean
```

Comment:
Is there a reason to manage differently isVscode & isTheia in the `send` and `on` methods ?
If not could we only keep `isVscode` ?